src/jsBundle.js

```diff
@@ -13,7 +13,7 @@ const installDependencies = require("./modules/install-dependencies");
 const createJavaScriptBundle = require("./modules/create-javascript-bundle");
 const { SolveFailure } = require("./modules/SolveFailure");
 const { UserException, FormatException } = require("./modules/HOME");
-
+const minifyBundle = require("./minify-javaScript");
```

Comment:
this file name is strange, a combo of camel and kebab. `minify-js` ?

Reply:
hmm:

```
 	create-entry-file-js.js 
	delete.js 
	jsBundle.js 	
	parse-modules-parameter.js 	
	update_origami_component_list.js 	
```
one snake, some kebab, one camel

Reply:
Done 👌